### PR TITLE
changes to risk form validation fixes #883

### LINKED ISSUE #883: There is no validation on the "Risks" field even if the "Yes" radio button is checked
**[Affected versions]:**
- Firefox Release 65.0 and above
- Chrome 72.0.3626.81

**[Affected Platforms]:** 	 	
- All Windows
- All Mac
- All Linux

**[Prerequisites]:**
- Have an experiment created on the "https://experimenter-app.dev.mozaws.net/" website.
- You are logged with your LDAP account.

**[Steps to reproduce]:**
1. Open the browser with the profile from prerequisites and navigate to "https://experimenter-app.dev.mozaws.net/".
2. Open the experiment from prerequisites and click the "Edit" button from the "Risks" section.
3. Delete the placeholder text from the "Risk" input field.
4. Click the "Save Draft and Finish" button and observe the behavior.

**[Expected result]:**
- An error message is displayed.

**[Actual result]:**
- The experiment is successfully saved.

**[Notes]:**
- This issue is not reproducible with the "Technical Risks Description" field.
- It seems that this issue is a regression caused by bug #829.
- Attached is a screen recording of the issue:
rec of the issue

### PR DESCRIPTION
**Pull request by tiftran**

### CODE REVIEW

File: app/experimenter/experiments/forms.py [hunk 1]
```diff
@@ -725,6 +725,21 @@ class Meta:
 
     def clean(self):
         cleaned_data = super().clean()
+        self._validate_risk(cleaned_data)
+        self._validate_risk_technical(cleaned_data)
+        return cleaned_data
+
+    def _validate_risk(self, cleaned_data):
+        if True in cleaned_data.values():
```
Comment (glasserc) [inline app/experimenter/experiments/forms.py]: Hmm! This is a little too clever for my taste. I'd probably prefer `if any([cleaned_data.risk_internal_only, cleaned_data.risk_partner_related, cleaned_data.risk_brand, ...])`.

**Comment by jaredkerim** [comment]
@tiftran Sorry for the hold up, I've looked at this but I think I need to talk to @shell1 about changes we're making to forms in general to figure out how this should look.  Sorry for the delay! -_-'

File: app/experimenter/templates/experiments/edit_risks.html [hunk 2]
```diff
@@ -86,14 +86,18 @@ <h4><span class="fas fa-info-circle"></span> Risks &amp; Testing</h4>
...
+        $('#risks_field .text-muted').hide();
+
+        if (hasRisk) {
           $('#risks_field').show();
         } else {
           $('#risks_field').hide();
         }
       }
-
+
```
Comment (glasserc) [inline app/experimenter/templates/experiments/edit_risks.html]: Whitespace change?

Comment (tiftran) [reply app/experimenter/templates/experiments/edit_risks.html]: let me fix that...

**Comment by tiftran** [comment]
Closing as discussed during standup with @jaredkerim  and @shell1 that behaviour of required has changed.

**Pull request closed without merging**